qemu/tests/block_copy.py

```diff
@@ -428,3 +428,44 @@ def verify_md5(self, file_name):
         if status != 0:
             raise error.TestFail("File %s changed, md5sum check output: %s" %
                                  (file_name, output))
+
+    def mount_disks(self):
+        """
+        Mount added disks inside guest.
+        """
+        os_type = self.params["os_type"]
+        if os_type == "linux":
```

Comment:
@CongLi Thanks, since it is a common function in block_copy, logically, I can not ensure the behavior from where call this function, should make sure this function only works for linux, so for windows just do nothing. I think that makes the function behave itself, which is logically correct.